packages/plugin-css/src/index.ts

```diff
@@ -345,88 +349,92 @@ export function transformStrokeStyle(value: ParsedStrokeStyleToken['$value']): s
   return String(value);
 }
 
-export function defaultTransformer(token: ParsedToken, {colorFormat = 'hex', mode, prefix}: {colorFormat: Options['colorFormat']; mode?: string; prefix?: string}): string | number | Record<string, string> {
+export function defaultTransformer(
+  token: ParsedToken,
+  tokens: ParsedToken[],
```

Comment:
Also here, too—didn’t notice this change to the `defaultTransformer()` function signature that was a breaking change for this plugin (because it taps into `transform()`). Again, there were no tests to catch this, so that’s a fault of the test suite! Just leaving notes.